src/scopes.rs

```diff
@@ -1106,6 +1124,67 @@ switch (foo) {
     assert!(array_object_method_scope.get_binding("d").is_some());
   }
 
+  #[test]
+  fn import_binding() {
+    let ast_parser = AstParser::new();
+    let syntax = swc_util::get_default_ts_config();
+
+    let source_code = r#"
+    import defaultExport1 from "module-name";
+    import * as namespaced1 from "module-name";
+    import { export1 } from "module-name";
+    import { export2 as alias1 } from "module-name";
+    import { export3 , export4 } from "module-name";
+    import { export5 , export6 as alias2} from "module-name";
+    import defaultExport2, { export7 } from "module-name";
+    import defaultExport3, * as namespaced2 from "module-name";
+    import "module-name";
```

Comment:
I think that this rule should also catch violations for bindings in `export { a, b, c } from "./asdf.ts"` - if I'm not mistaken there's no separate rule for exports

Reply:
in export-from  declerations are not new bindings in the working script.
```ts
// export
export { a } from './a.ts';
a = 2; // 2:1 error 'a' is not defined. (no-undef)

// import
import { a } from './a.ts';
a = 2; // 2:1 error 'a' is read-only. (no-import-assign)
```

did I miss something? or a more specific example, please?

Reply:
Oooops, you're absolutely right @disizali, my bad